app/javascript/app/pages/home/home-component.jsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import Intro from 'components/intro';
+import Section from 'components/section';
+import Search from 'components/search';
+import Icon from 'components/icon';
+import Button from 'components/button';
+import Dropdown from 'components/dropdown';
+import ReactPlayer from 'react-player';
+import cx from 'classnames';
+
+import cwLogo from 'assets/icons/cw-logo-white.svg';
+import fullscreen from 'assets/icons/map-fullscreen.svg';
+import homeOneBg from 'assets/backgrounds/home_bg_1';
+import homeTwoBg from 'assets/backgrounds/home_bg_2';
+import homeTwoImage from 'assets/backgrounds/home_image_2';
+import homeThreeBg from 'assets/backgrounds/home_bg_3';
+import homeThreeImage from 'assets/backgrounds/home_image_3';
+
+import styles from './home-styles.scss';
+
+const Home = () =>
+  (<div>
+    <Section className={styles.homeOne} backgroundImage={homeOneBg}>
+      <div className={styles.column}>
+        <Icon icon={cwLogo} className={styles.cwLogo} theme={styles} />
```

Comment:
Why do we need to have all the styles of the homePage in the theme prop to the icon component? I also think they are not being used.